Main.cpp

```diff
@@ -69,7 +69,21 @@ int appLauncherMain(int argc, char** argv)
 #endif
         );
 
-  QFileInfo launcherFile(QDir::current(), QString(argv[0]));
+  QString executableName = argv[0];
```

Comment:
Would be nice to add a test for this. A test named `Testing/Launcher/AppLauncher-TestWithoutExeExtension.cmake` (similar to [this](https://github.com/commontk/AppLauncher/blob/master/Testing/Launcher/AppLauncher-TestCurrentDirectory.cmake) one) could be added